cardano-api/src/Cardano/Api/LedgerState.hs

```diff
@@ -892,12 +940,11 @@ tickThenReapplyCheckHash
     -> Shelley.LedgerState
         (HFC.HardForkBlock
             (Consensus.CardanoEras Shelley.StandardCrypto))
-    -> Either Text (Shelley.LedgerState
-        (HFC.HardForkBlock
-            (Consensus.CardanoEras Shelley.StandardCrypto)))
+    -> Either Text LedgerStateEvents
 tickThenReapplyCheckHash cfg block lsb =
   if Consensus.blockPrevHash block == Ledger.ledgerTipHash lsb
-    then Right $ Ledger.tickThenReapply cfg block lsb
```

Comment:
I just pushed a refined interface to PR input-output-hk/ouroboros-network#3292

I think this should now just be `... $ Ledger.tickThenReapplyLedgerResult cfg block lsb`, no need for `runIdentity` or `runLedgerT`.